pkg/cvo/sync_worker.go

```diff
@@ -148,11 +148,14 @@ type SyncWorker struct {
 
 	// updated by the run method only
 	payload *payload.Update
+
+	excludeFilter []string
```

Comment:
why is this called a filter but is a string slice? this doesn't really provide context on how to use this to filter.

Reply:
It's just a slice of regular expressions. I can add a comment explaining what they are.

Reply:
maybe plural Filters?